QtQuick1 does no longer get released since 5.5.1.

Pull in a patch from git to make it build against the changed api in newer
Qt versions.

From 26229cfa0b729313893af5674d604e8692dbb946 Mon Sep 17 00:00:00 2001
From: Richard Moe Gustavsen <richard.gustavsen@theqtcompany.com>
Date: Tue, 26 Apr 2016 15:08:34 +0200
Subject: [PATCH] QDeclarativeTextInput: update API to use
 setBlinkingCursorEnabled

QWidgetLineControl::setCursorBlinkPeriod was renamed to
QWidgetLineControl::setBlinkingCursorEnabled in qtbase:1b5bc97.
This patch will update QDeclarativeTextInput to use the new API.

Change-Id: If88c0d44f2ab29c9bc00f5a0226a28d45976ed43
Reviewed-by: Simon Hausmann <simon.hausmann@theqtcompany.com>
URL: https://codereview.qt-project.org/#/c/157118
---
 src/declarative/graphicsitems/qdeclarativetextinput.cpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/declarative/graphicsitems/qdeclarativetextinput.cpp b/src/declarative/graphicsitems/qdeclarativetextinput.cpp
index 3ff75fd..e39d1af 100644
--- src/declarative/graphicsitems/qdeclarativetextinput.cpp
+++ src/declarative/graphicsitems/qdeclarativetextinput.cpp
@@ -514,7 +514,7 @@ void QDeclarativeTextInput::setCursorVisible(bool on)
     if (d->cursorVisible == on)
         return;
     d->cursorVisible = on;
-    d->control->setCursorBlinkPeriod(on?QApplication::cursorFlashTime():0);
+    d->control->setBlinkingCursorEnabled(on);
     QRect r = d->control->cursorRect();
     if (d->control->inputMask().isEmpty())
         updateRect(r);
--
2.8.1

